src/vma-xlio-redirect.h

```diff
@@ -40,6 +40,9 @@
 #ifdef __FreeBSD__
 #include <netinet/in.h>
 #include <stdio.h>
+#elif __APPLE__
+#include <netinet/in.h>
+#include <stdio.h>
```

Comment:
please consider using one section in case no difference for `linux` and `macos` as
`#elif defined(__linux__) || defined(__APPLE__) `